app/views/catalog/_document_list.html.erb

```diff
@@ -1,5 +1,11 @@
 <% # container for all documents in index list view -%>
 <% view_config = local_assigns[:view_config] || blacklight_config&.view_config(document_index_view_type) %>
 <div id="documents" class="documents-<%= view_config&.key || document_index_view_type %>">
-  <%= render documents, as: :document, view_config: view_config %>
+  <% view_config.document_component.with_collection(documents, counter_offset: (@response&.start || 0)).each do |component| %>
```

Comment:
why is this not passing a presenter?  It used to when it was here: https://github.com/projectblacklight/blacklight/pull/2698/files#diff-8225b3193b5c90c9d0e11c0ab59b930bcd01e527430e0a1fc795093271946cc5L3

Reply:
the presenter it passed before reproduced the default logic for the presenter in the component; if we are committing to having document component wrap the presenter, we would change this call but also probably remove the  presenter-default logic in the component; this seemed like the smaller change.

Reply:
I wonder if we can hash this out on the next call.  I feel like making the component wrap the presenter, and have no knowledge of the document (fewer dependencies) is the better design.

Reply:
I don't disagree with that - I might suggest something similar to that `with_individual` method I removed in that case, maybe just to build a splattable options hash-- Just because it feel weird to say that DocumentPresenter's `document:` param should be a presenter. I just didn't want to be presumptuous here outside the scope of using unsurprising collection behaviors for the components.

Reply:
Like, I would be totally happy to open a ticket saying `DocumentComponent should expect a document presenter, not a SolrDocument` and do that next! It would let us delete a good amount of code in the DocumentComponent.